README.md

```diff
@@ -113,6 +113,127 @@ $ ./gradlew clean assembleRelease test
 $ ./gradlew clean assemble test
 ~~~
 
+### Branching/Merging
+
+We use [git flow](https://nvie.com/posts/a-successful-git-branching-model/) as our
+basis for branching and creating releases. We highly recommend installing
+[Git Flow AVH Edition](https://github.com/petervanderdoes/gitflow-avh) to
+automate some of the work of branching and tagging. Using `gitflow-avh`
+is not required, but by automating the underlying repository operations,
+it eliminates the possibility of making mistakes, and keeps the various
+branches consistent.
+
+### Releasing
```

Comment:
Having this documented is great! What would you think about moving it into a separate `RELEASING.md` doc and linking to it from here?

Reply:
Yep, no problem with that. Whatever gets this into a repository somewhere. :wink: